### PR TITLE
add Game Tests

### PR DESCRIPTION
**Pull request by DonnA98**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java [hunk 1]
```diff
@@ -85,6 +97,19 @@ public void testCanParseAnyDifficulty() {
...
+    public void testGameWithPlayer() {
+        uuid = "newUuid";
+        assertTrue(game.withPlayer(uuid).getPlayersUuid().contains(uuid));
+    }
+
+    @Test
+    public void testRemovePlayer() {
+        uuid = "newUuid2";
+        game.withPlayer(uuid);
+        assertTrue(!game.removePlayer(uuid).getPlayersUuid().contains(uuid));
```
Comment (rykeuse) [inline app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java]: use `assertFalse`

File: app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java [hunk 2]
```diff
@@ -46,18 +50,26 @@
     public final IntentsTestRule<CreateGameActivity> intentsTestRule = new IntentsTestRule<>(
             CreateGameActivity.class);
     private Resources systemResources;
+    private Game game;
+    private String uuid;
 
     @Before
-    public void setSystemResources() {
+    public void setVariables() {
```
Comment (rykeuse) [inline app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java]: Why did you change this?

Comment (DonnA98) [reply app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java]: because systemResources is no longer the only variable that is initialized
